README.md

```diff
@@ -424,4 +425,4 @@ Checking the company's Tech stack through [Stackshare](https://stackshare.io/) m
 | [Hostelworld](https://www.hostelworld.com/) [:rocket:](https://www.hostelworldgroup.com/careers) | World's leading hostel-focused online booking platform. | `Porto` |
 | [Ryanair](https://www.ryanair.com/pt/pt) [:rocket:](https://careers.ryanair.com/jobs/?ryanair-jobs-department=21233) | Ryanair is an Irish ultra low-cost carrier. | `Lisboa` `Porto`|
 | [TripAdvisor](https://www.tripadvisor.com) [:rocket:](https://careers.tripadvisor.com) | The largest "social travel website" in the world. | `Lisboa` |
-| [Vision-Box](https://www.vision-box.com/) [:rocket:](https://visionbox.teamtailor.com/jobs) | Biometric solutions for automated border control, identity management, and security. | `Lisboa` |
\ No newline at end of file
+| [Vision-Box](https://www.vision-box.com/) [:rocket:](https://visionbox.teamtailor.com/jobs) | Biometric solutions for automated border control, identity management, and security. | `Lisboa` |
```

Comment:
Added empty line at the end of file, good :+1: 

( I could not see this change properly through the GitHub Web UI :shrug: )